docs/extensions/abbreviations.md

```diff
@@ -46,7 +51,20 @@ Usage
 See [Extensions](index.md) for general extension usage. Use `abbr` as the name
 of the extension.
 
-This extension does not accept any special configuration options.
+The following options are provided to configure the output:
+
+* **`use_last_abbr`**:
```

Comment:
Docs need updated to reflect latest changes.

Reply:
Updated. Let me know if more details are needed for the `glossary` option and functions.